### PR TITLE
[GROW-575] Add channelUpgradeModal segment tracking

### PR DESCRIPTION
**Pull request by jacobchadwell**
Replace useTrackPageViewed event with useConnectChannelUpgradeModalViewed

### CODE REVIEW

**Comment by daisymarie128** [comment]
Awesome work @jacobchadwell , I just left 2 small changes I think we need to add. But everything is look great! 🙌

**Review by daisymarie128 — APPROVED** [review]

**Comment by daisymarie128** [comment]
@jacobchadwell Great work! 🎉  I think the Cypress action might need a rerun though to get it passing 😊

File: packages/app-shell/src/components/Modal/modals/ConnectChannelUpgrade/components/CardBody.jsx [hunk 1]
```diff
@@ -51,11 +51,10 @@ const CardBody = ({
   });
 
   useEffect(() => {
-    useTrackPageViewed({
```
[the remark this answers is not in the document]
Comment (jacobchadwell) [reply packages/app-shell/src/components/Modal/modals/ConnectChannelUpgrade/components/CardBody.jsx:54]: @daisymarie128 thanks so much for the context! 😄 I'll be curious as well from the team how we use these!

File: packages/app-shell/src/components/Modal/modals/ConnectChannelUpgrade/components/CardBody.jsx [hunk 2]
```diff
@@ -51,11 +51,10 @@ const CardBody = ({
...
+    useConnectChannelUpgradeModalViewed({
       payload: {
-        name: 'billingIncreaseSubscriptionQuantity',
-        title: 'channelConnectionSuccess',
-        cta,
+        currentPlan: formatCTAString(`${planId} ${planCycle}`),
+        cta
```
[the remark this answers is not in the document]
Comment (jacobchadwell) [reply packages/app-shell/src/components/Modal/modals/ConnectChannelUpgrade/components/CardBody.jsx]: Awesome, made the updates!

**Pull request merged**